release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -129,6 +146,21 @@ the {product-title} web console.
 The `KubeletTooManyPods` alert now uses the Pod capacity metric as a threshold
 instead of a fixed number.
 
+[id="ocp-4-3-monitor-your-own-services"]
+==== Monitor your own Services (Technology Preview)
+
+The existing monitoring stack can be extended so you can configure monitoring
+for your own Services.
+
+[id="ocp-4-3-query-browser-available-throught-devexp-console"]
+==== Query browser available through the DevExp console (Technology Preview)
+
+Querying metrics is now available through the developer perspective inside the
```

Comment:
Is the _developer perspective_ what the DevExp console is? Or is it the developer tab available in the UI?

Reply:
It's referring to the Developer tab in the UI. I'll make this more clear.